### PR TITLE
docs(cdevents-notification): user guide to configure CDEvents notification

### PR DESCRIPTION
**Pull request by rjalander**
Documentation changes to configure CDEvents type notifications.
CDEvents Notification is implemented as part of feature accepted [cdevents-spinnaker](https://github.com/spinnaker/governance/blob/master/rfc/cdevents-spinnaker.md)

### CODE REVIEW

File: content/en/docs/setup/other_config/features/notifications/index.md [hunk 1]
```diff
@@ -123,6 +124,23 @@ hal config notification twilio enable
...
+## CDEvents
+
+To enable CDEvents notification support, add the following statement to the `echo-local.yml` file:
+
+```yaml
+cdevents:
+  enabled: true
+```
+
+Spinnaker supports sending CDEvents to [Message Broker](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessageBroker.html)
```
Comment (jasonmcintosh) [inline content/en/docs/setup/other_config/features/notifications/index.md:137]: "to a" vs. "to"

Comment (jasonmcintosh) [reply content/en/docs/setup/other_config/features/notifications/index.md:137]: Are there any docs on configuring the target destination though as part of this?

Comment (jasonmcintosh) [reply content/en/docs/setup/other_config/features/notifications/index.md:137]: Sorry rephrase:  Can you maybe mention "The target broker is defined in the notification trigger itself".  I WOULD be... cautious given that this is an arbitrary publish of payload data to a URL, similar to how webhooks operate.

Comment (rjalander) [reply content/en/docs/setup/other_config/features/notifications/index.md:137]: Thank you @jasonmcintosh for reviewing this PR.
May be I can share the configuration for [Knative Eventing](https://knative.dev/docs/getting-started/) as a Message Broker

**Review by jasonmcintosh — APPROVED** [review]

**Pull request merged**